tests/cpp/cell/test_gemm.cu

```diff
@@ -114,22 +113,20 @@ struct TestTraits {
     static constexpr int kWarpPerCol = tl::num_cols<WarpLayout>;
 
     /// == 2. configurate tile transfer between global and shared using CuTe ==
-    using LoadSharedA =
-        traits::G2S2DCopyTraits<Element, kM, kK, kM, kK, kThreads>;
-    using LoadSharedB =
-        traits::G2S2DCopyTraits<Element, kN, kK, kN, kK, kThreads>;
-    // transfer operand C from shared memory to global memory
-    using StoreSharedC =
-        traits::S2G2DCopyTraits<ElementAcc, kM, kN, kM, kN, kThreads>;
+    using GlobalA = GlobalTile<Element, tl::RowMajor<kM, kK>>;
+    using SharedA = SharedTile<Element, tl::RowMajor<kM, kK>>;
+    using LoadSharedA = GlobalToSharedLoader<SharedA, WarpLayout>;
+
+    using GlobalB = GlobalTile<Element, tl::RowMajor<kN, kK>>;
+    using SharedB = SharedTile<Element, tl::RowMajor<kN, kK>>;
+    using LoadSharedB = GlobalToSharedLoader<SharedB, WarpLayout>;
 
     /// === 3. configurate tile transfer between shared and register loader ===
     // shared tile for operand A
-    using SharedA = SharedTile<Element, tl::RowMajor<kM, kK>>;
     using TileIteratorA = TileIterator<SharedA, TileShape<kM, kChunkK>>;
-
     // shared tile for operand B
-    using SharedB = SharedTile<Element, tl::ColMajor<kK, kN>>;
-    using TileIteratorB = TileIterator<SharedB, TileShape<kChunkK, kN>>;
+    using SharedB2 = SharedTile<Element, tl::ColMajor<kK, kN>>;
```

Comment:
Defining two layouts for B on shared memory is an implementing issue. Because the current GlobalToShared is only partially specialized for RowMajor, during the loading process from global to shared memory, operand B is interpreted as an equivalent RowMajor layout.

Reply:
Okay, we can address this issue in a future PR.

Reply:
This actually must be addressed. I am working on figuring out how to enable swizzled for operand B.